cms/templates/home_page.html

```diff
@@ -18,35 +18,110 @@ <h3>{{ page.hero_subtitle }}</h3>
     </div>
   </div>
 </div>
-<div class="course-section">
-  <div class="container">
-    {% if product_cards_section_title %}
-      <h1>{{ product_cards_section_title }}</h1>
-    {% endif %}
-    <ul class="course-listing card-listing">
-      {% for product in products %}
-      <li>
-        <div class="course-card highlight-card clickable">
-          <a href="{{ product.url_path }}" class="course-card-link">
-            <img src="{% feature_img_src product.feature_image %}" alt="">
-            <div class="course-info">
-              <h2 class="course-card-title">
-                  {{ product.title }}
-              </h2>
-              <div class="course-card-description">
-                {{ product.description | richtext }}
+{% if show_new_featured_carousel %}
+  <div class="featured-product-section-container container">
+    <div class="featured-product-section carousel slide" id="featuredProductCarousel">
+      <div class="featured-product-header">
+        {% if product_cards_section_title %}
+          <div class="featured-product-header-title">{{ product_cards_section_title }}</div>
+        {% endif %}
+        <div class="featured-product-arrows">
+          <a href="#featuredProductCarousel" role="button" data-bs-slide="prev">
+            <img class="arrow prev-arrow" src="/static/images/featured-products-left-arrow.svg" alt="Previous Arrow" />
+          </a>
+          <a href="#featuredProductCarousel" role="button" data-bs-slide="next">
+            <img class="arrow next-arrow" src="/static/images/featured-products-right-arrow.svg" alt="Next Arrow" />
+          </a>
+        </div>
+      </div>
+      <div class="featured-product-listing">
+        <div class="carousel-inner" role="listbox">
+          {% for product in products %}
+          <div class="carousel-item{% if forloop.counter == 1 %} active{% endif %} ">
+            <div class="card featured-product-card">
+              <a href="{{ product.url_path }}" class="featured-product-card-link">
+              <div class="featured-product-thumb">
+                <img src="{% feature_img_src product.feature_image %}" alt="" />
+                {% if product.is_program and product.program_type %}
+                  <span class="badge badge-program-type">{{ product.program_type }}</span>
+                {% endif %}
+              </div>
+              <div class="featured-product-info card-body">
+                {% if product.start_date and not product.is_program and show_new_featured_carousel %}
+                  <p class="date">
+                    Starts {{ product.start_date | date:"F j, Y" }}
+                  </p>
+                {% elif not product.is_program %}
+                  <p class="date">
+                    Start Anytime
```

Comment:
Nevermind. I guess it makes sense to say Start Anytime if all existing runs are expired

Reply:
I would add `not product.start_date` to be clear.

Reply:
so not is_program and not  start_date? without new_featured carousel the prior if statement is just "if start date and is not program" - this is "else if" so the not start_date is implied, we just need to make sure it's not a program. Not sure what we gain.

Reply:
I think its because the previous if `product.start_date and not product.is_program and show_new_featured_carousel`  made this unclear, but if you remove `show_new_featured_carousel`, then yes no need to update here. It took me a sec to realize what this implies when I first look at it.

Reply:
Modified